app/controllers/sessions_controller.rb

```diff
@@ -15,4 +15,9 @@ def create
     end
   end
 
+  def destroy
+    session[:user_id] = nil
+    redirect_to '/'
```

Comment:
A good practice in Rails is to use the generated `*_paths` instead of string literals. You can list the available routes using either `bin/rake routes` or navigating to http://localhost:3000/routes (my favorite, which offers a search option).

For this case you can use the suggested snippet below:
```suggestion
    redirect_to root_path
```